docs/cody/capabilities/agentic-chat.mdx

```diff
@@ -0,0 +1,105 @@
+
+### Getting Agentic Chat access for Enterprise customers
+
+Enterprise customers can get access by upgrading their client (VS Code or JetBrains) to the latest version of the plugin and enabling the following feature flags on their Sourcegraph Instance:
+
+- `deep-cody` to get access to the feature
+- `deep-cody-shell-context` to allow [terminal access](#terminal-commands)
+- Users will also need to manually enable access to the terminal commands on the client's `settings.json` file i.e., set `"cody.agentic.context.experimentalShell": true`
+
+## What can Agentic Chat do?
+
+Agentic Chat can help you with the following:
+
+### Tool Usage
+
+It has access to a suite of tools for retrieving relevant context. These tools include:
+
+- **Search Tool**: Searches the codebase using keywords extracted from your query
+- **File Tool**: Retrieves the content of specific files in your codebase
+- **CLI Tool**: Executes terminal commands to gather system or project-specific information
+- **Memory Tool**: Allows the agent to create personal notes that can be used across chat sessions
+- **Web Browser Tool (via OpenCtx)**: Searches the web for live context
+- **Linear Issue Tool (via OpenCtx)**: Integrates with Linear issue tracking when the provider is configured
+
+It integrates seamlessly with external services, such as web content retrieval and issue tracking systems, using OpenCtx providers. To learn more, [read the OpenCtx docs](/cody/capabilities/openctx).
+
+Memory allows the Agentic Chat to learn and remember your preferences, enabling a more personalized experience. You can ask it to **remember** specific details for future interactions.
+
+<Callout type="info">Tool calling is not supported on the Web. It only works with VS Code and JetBrains editor extensions.</Callout>
+
+## Terminal commands
+
+Agentic Chat can use the CLI Tool to execute shell commands and gather context from your terminal. Its ability to execute terminal commands enhances its context-gathering capabilities. However, it’s essential to understand that any information accessible via your terminal could potentially be shared with the LLM. It's recommended not to request information that you don't want to share. Here's what you should consider:
+
+- **Trusted workspaces only**: Commands can only be executed within trusted workspaces with a valid shell
+- **Potential data sharing**: Any terminal-accessible information may be shared with the LLM
+- **Allow list configuration**: This feature is disabled if the allow list is empty. By default, the allow list is set to ['*'], which allows all commands. For enhanced security, use specific command prefixes instead of the wildcard *.
+
+Commands are generated by the agent/LLM based on your request. Avoid asking it to execute destructive commands if you do not want them to run.
+
+### Configure shell command execution
+
+You can configure shell command execution via the `cody.agentic.context` setting in your editor:
```

Comment:
```suggestion
You can configure terminal access in the chat UI, under the agentic context settings.
```

<img width="664" alt="image" src="https://github.com/user-attachments/assets/0d53e82c-50d1-46ab-a9f5-919f8953995b" />

Reply:
@abeatrix where will this option live with the new UI?